src/components/Dashboard/Header/index.tsx

```diff
@@ -1,34 +1,49 @@
 import React from 'react';
 
 import { ReactComponent as NimbleLogoWhite } from 'assets/images/icons/nimble-logo-white.svg';
+import Shimmer from 'components/Shimmer';
 
 interface DashboardHeaderProps {
   dateTime: string;
   daysAgo: string;
   profileUrl: string;
+  shouldShowShimmer?: boolean;
   children: React.ReactNode;
   'data-test-id'?: string;
```

Comment:
could we move the `data-test-id` inside the component, just like `DashboardContent`?